docs/pages/versions/unversioned/sdk/gl-view.md

```diff
@@ -156,38 +89,6 @@ Since the WebGL API is quite low-level, it can be helpful to use higher-level gr
 
 Any WebGL-supporting library that expects a [WebGLRenderingContext](https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.14) could be used. Some times such libraries assume a web JavaScript context (such as assuming `document`). Usually this is for resource loading or event handling, with the main rendering logic still only using pure WebGL. So these libraries can usually still be used with a couple workarounds. The Expo-specific integrations above include workarounds for some popular libraries.
 
-## WebGL API
-
-Once the component is mounted and the OpenGL ES context has been created, the `gl` object received through the `onContextCreate` prop becomes the interface to the OpenGL ES context, providing a WebGL API. It resembles a [WebGL2RenderingContext](https://www.khronos.org/registry/webgl/specs/latest/2.0/#3.7) in the WebGL 2 spec. However, some older Android devices may not support WebGL2 features. To check whether the device supports WebGL2 it's recommended to use `gl instanceof WebGL2RenderingContext`.
-An additional method `gl.endFrameEXP()` is present which notifies the context that the current frame is ready to be presented. This is similar to a 'swap buffers' API call in other OpenGL platforms.
-
-The following WebGL2RenderingContext methods are currently unimplemented:
-
-- `getFramebufferAttachmentParameter()`
-- `getRenderbufferParameter()`
-- `compressedTexImage2D()`
-- `compressedTexSubImage2D()`
-- `getTexParameter()`
-- `getUniform()`
-- `getVertexAttrib()`
-- `getVertexAttribOffset()`
-- `getBufferSubData()`
-- `getInternalformatParameter()`
-- `renderbufferStorageMultisample()`
-- `compressedTexImage3D()`
-- `compressedTexSubImage3D()`
-- `fenceSync()`
-- `isSync()`
-- `deleteSync()`
-- `clientWaitSync()`
-- `waitSync()`
-- `getSyncParameter()`
-- `getActiveUniformBlockParameter()`
-
-The `pixels` argument of [`texImage2D()`](https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/texImage2D) must be `null`, an `ArrayBuffer` with pixel data, or an object of the form `{ localUri }` where `localUri` is the `file://` URI of an image in the device's file system. Thus an `Asset` object could be used once `.downloadAsync()` has been called on it (and completed) to fetch the resource.
-
-For efficiency reasons the current implementations of the methods don't perform type or bounds checking on their arguments. So, passing invalid arguments could cause a native crash. We plan to update the API to perform argument checking in upcoming SDK versions. Currently the priority for error checking is low since engines generally don't rely on the OpenGL API to perform argument checking and, even otherwise, checks performed by the underlying OpenGL ES implementation are often sufficient.
-
 ## Integration with Reanimated worklets
 
 To use this API inside Reanimated worklet you need to pass the GL context ID to the worklet and recreate the GL object like in the example bellow.
```

Comment:
```suggestion
To use this API inside Reanimated worklet, you need to pass the GL context ID to the worklet and recreate the GL object like in the example below.
```

Just a small typo fix and some grammar changes(I know this is unrelated to the changes you pushed 😄)